Asterisk.2013/Asterisk.NET/Manager/ManagerConnection.cs

```diff
@@ -15,1566 +15,917 @@
-			Helper.RegisterEventHandler(registeredEventHandlers, 62, typeof(DisconnectEvent));
-			Helper.RegisterEventHandler(registeredEventHandlers, 62, typeof(ReloadEvent));
-			Helper.RegisterEventHandler(registeredEventHandlers, 62, typeof(ShutdownEvent));
-
-			Helper.RegisterEventHandler(registeredEventHandlers, 63, typeof(BridgeEvent));
-			Helper.RegisterEventHandler(registeredEventHandlers, 64, typeof(TransferEvent));
-			Helper.RegisterEventHandler(registeredEventHandlers, 65, typeof(DTMFEvent));
-
-			Helper.RegisterEventHandler(registeredEventHandlers, 70, typeof(VarSetEvent));
-			Helper.RegisterEventHandler(registeredEventHandlers, 80, typeof(AGIExecEvent));
-
-			Helper.RegisterEventHandler(registeredEventHandlers, 81, typeof(ConfbridgeStartEvent));
-			Helper.RegisterEventHandler(registeredEventHandlers, 82, typeof(ConfbridgeJoinEvent));
-			Helper.RegisterEventHandler(registeredEventHandlers, 83, typeof(ConfbridgeLeaveEvent));
-			Helper.RegisterEventHandler(registeredEventHandlers, 84, typeof(ConfbridgeEndEvent));
-			Helper.RegisterEventHandler(registeredEventHandlers, 85, typeof(ConfbridgeTalkingEvent));
-
-            Helper.RegisterEventHandler(registeredEventHandlers, 86, typeof(FailedACLEvent));
-
-            Helper.RegisterEventHandler(registeredEventHandlers, 87, typeof(AttendedTransferEvent));
-            Helper.RegisterEventHandler(registeredEventHandlers, 88, typeof(BridgeCreateEvent));
-            Helper.RegisterEventHandler(registeredEventHandlers, 89, typeof(BridgeDestroyEvent));
-            Helper.RegisterEventHandler(registeredEventHandlers, 90, typeof(BridgeEnterEvent));
-            Helper.RegisterEventHandler(registeredEventHandlers, 91, typeof(BridgeLeaveEvent));
-            Helper.RegisterEventHandler(registeredEventHandlers, 92, typeof(BlindTransferEvent));
-            Helper.RegisterEventHandler(registeredEventHandlers, 93, typeof(DialBeginEvent));
-            Helper.RegisterEventHandler(registeredEventHandlers, 94, typeof(DialEndEvent));
-            Helper.RegisterEventHandler(registeredEventHandlers, 95, typeof(QueueCallerJoinEvent));
-            Helper.RegisterEventHandler(registeredEventHandlers, 96, typeof(QueueCallerLeaveEvent));
-            Helper.RegisterEventHandler(registeredEventHandlers, 97, typeof(QueueMemberPauseEvent));
+        {
+            callerThread = Thread.CurrentThread;
+
+            socketEncoding = Encoding.ASCII;
+
+            responseHandlers = new Dictionary<int, IResponseHandler>();
+            pingHandlers = new Dictionary<int, IResponseHandler>();
+            responseEventHandlers = new Dictionary<int, IResponseHandler>();
+            registeredEventClasses = new Dictionary<int, ConstructorInfo>();
+
+            Helper.RegisterBuiltinEventClasses(registeredEventClasses);
+
+            registeredEventHandlers = new Dictionary<int, Action<ManagerEvent>>();
+
+            #region Event mapping table

```

Comment:
Can we clarify, this is a breaking change still? Event signatures have changed? Not had time to pull it down and test, so could do with some details from you please.